internal/instruments.go

```diff
@@ -0,0 +1,44 @@
+package internal
+
+import (
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/metric"
+	"go.opentelemetry.io/otel/api/unit"
+	"log"
+)
+
+type openTelemetryInstrumentation struct {
+	// Count of write commands performed
+	WriteCount metric.Int64Counter
+	// Count of new connections
+	NewConnectionsCount metric.Int64Counter
+}
+
+var Instruments = initInstruments()
+
+func initInstruments() *openTelemetryInstrumentation {
+	meter := global.Meter("github.com/go-redis/redis")
+
+	writeCount, err := meter.NewInt64Counter("redis.num_writes",
```

Comment:
I would leave out the prefix `num`. It is clear by the instrument type this is counting something and a metric name of `writes` will describe what that is.

```suggestion
	writeCount, err := meter.NewInt64Counter("redis.writes",
```